### ISSUE TITLE
Protein Family names not being displayed on Protein Family page

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#1313]
I am comparing 3 groups and I chose PLFams.  When the page loaded (FINALLY loaded), some of the protein families do not have names:
<img width="810" alt="screen shot 2016-11-29 at 1 33 57 pm" src="https://cloud.githubusercontent.com/assets/11475316/20723484/bda6559c-b638-11e6-88eb-ef2015aa6ce0.png">

**Comment by mshukla1** [comment]
This is data problem. I am working on it. Should be fixed soon.

**Comment by ARWattam** [comment]
there are some issues with displaying the family id as well.  Look at the second row down.
<img width="811" alt="screen shot 2016-11-29 at 1 58 24 pm" src="https://cloud.githubusercontent.com/assets/11475316/20724401/095008be-b63c-11e6-9c75-23e5407817ad.png">

**Comment by mshukla1** [comment]
is this for public or private genome? can you send me the url, si I can verify when fixed?

**Comment by ARWattam** [comment]
I was comparing groups using the protein family sorter.  One is a private genome, the rest are public.

----- Original Message -----

**Comment by mshukla1** [comment]
Try it again, and see if the blank product names are gone now.

**Comment by ARWattam** [comment]
Still getting the error:
<img width="1110" alt="screen shot 2016-11-29 at 3 00 52 pm" src="https://cloud.githubusercontent.com/assets/11475316/20726705/b05a54d6-b644-11e6-837e-e483f9de4072.png">

Also seeing a protein family named 12
<img width="862" alt="screen shot 2016-11-29 at 3 01 47 pm" src="https://cloud.githubusercontent.com/assets/11475316/20726729/ce4b671e-b644-11e6-9b2c-49e2ba4fc157.png">

**Comment by ARWattam** [comment]
Also some oddly named FIGFams
<img width="797" alt="screen shot 2016-11-29 at 3 05 18 pm" src="https://cloud.githubusercontent.com/assets/11475316/20726878/5d340a9e-b645-11e6-9197-16b499eb2c81.png">

**Comment by hyoo** [comment]
Per Bob's comment,
"Veronika fixed these assignments in September and they are gone in the new families."

#1046

**Comment by mshukla1** [comment]
The problems with the PL and PGfams have been fixed. 

There is not much we can do about the odd figfam product names as they are no longer maintained / updated. Also, they do not get synced with the annotation updates, so the function of the figfams might be inconsistent with the function of its members. 

When we released Pattyfams, we kept figfams to give people time to transition to new families. Given it has been more than a year now, I propose that we get rid of figfams in the next release.
